policyengine_us/parameters/gov/states/ri/tax/income/rate.yaml

```diff
@@ -5,7 +5,14 @@ metadata:
   rate_unit: /1
   period: year
   label: Rhode Island income tax rate
+  uprating:
+    parameter: gov.irs.uprating
+    rounding:
+      type: downwards
+      interval: 50
```

Comment:
we need to add this metadata piece to each individual threshold (besides 0) as this structure will also uprate the rate, which should stay the same